src/asciidoc/docs/develop/app-dev-process.adoc

```diff
@@ -0,0 +1,206 @@
+
+. In your backend directory, install or update the @withkoji/core package.
++
+[source, Bash]
+npm install @withkoji/core
+
+. In your backend code, import the package and your app's configuration data.
++
+[source,Javascript]
+----
+import { KojiBackend } from '@withkoji/core';
+
+import kojiConfig from '../../koji.json';
+----
+
++
+TIP: To initialize the core package in your backend, the `koji.json` file must be bundled with the backend service during deployment of your app.
+Many frameworks won't bundle this file by default because it is not in the same folder as the backend code.
+To solve this problem, you can use a tool such as https://www.npmjs.com/package/babel-plugin-inline-json-import[babel-plugin-inline-json-import].
+
+. Use `KojiBackend.middleware` to initialize the package with your app's configuration data.
++
+The middleware manages the scope for backend functionality, so that your code can support your initial app along with all of the custom versions.
+It works by parsing the request headers that the Koji platform uses to pass version-specific data to the backend.
++
+You must apply this middleware before handling any routes.
+The middleware will transform `res.locals`, and then modules in KojiBackend will accept `res` as a configuration parameter.
+For example, here is an implementation with a traditional Express application.
++
+[source,Javascript]
+----
+include::{includespath}/backend-core.js[]
+----
+
+. Implement the desired core functionality.
+For example, you can implement a Koji database to collect information in the app.
++
+Learn more: <<core-backend-overview#>>, <<add-service#>>, <<koji-database#>>
+
+== [.callout3]#Deploy your app#
+
+When you're finished developing your app, you can deploy it to the Koji platform.
+Deployed apps are accessible at a permanent production URL.
+
+. Deploy your app from your development environment.
+
+* If you're working in the Koji code editor, you can publish your app from there.
++
+Learn more: <<editor#_publishing_your_koji_app, Publishing your Koji app>>
+
+* If you're working outside the editor, you must push your changes to your project’s origin repository.
++
+Learn more: <<publish-locally-developed#>>
+
++
+After you publish the app, the Koji profiler checks for errors, generates metadata about it, and, if it is error-free, enables the platform entitlements.
+
+. To verify that your latest build was deployed, go to your https://withkoji.com/developer/projects[developer portal] and select the app.
+Then, click btn:[View Version History].
++
+Learn more: <<app-versions#_viewing_version_history, Viewing version history>>
+
+TIP: If you update your app and deploy it again, the new version of the app replaces the existing version at the production URL.
+However, your app listing in the App Store is not automatically updated.
+This feature lets you release test versions of updates without affecting a currently listed version in the App Store.
```

Comment:
While this is where we're headed, it is not technically the case today. The most recent published version of an app is the one that is available on the App Store. Versions within the App Store are used only for changes to metadata.

Reply:
Thanks! Removing the last two sentences for now.